README.md

```diff
@@ -73,6 +73,9 @@ plugin allows an OP to authenticate against a Relying Party using Self-Issued Op
 Connect for Verifiable Presentations, with the help of
 our [Presentation-Exchange library](https://github.com/Sphereon-Opensource/pe-js).
 
+## WACI PEx
+The [waci-pex](./packages/waci-pex/README.md) is a Veramo plugin to generate QR Code using a [WACI Presentation Exchange](https://identity.foundation/waci-presentation-exchange/#step-1-generate-qr-code) spec.
```

Comment:
"is a Veramo plugin to generate a QR Code using the"

Reply:
updated

Reply:
1 in the plugin readme is still not addressed.

plus here it now says "a [WACI Presentation Exchange] specification", which implies there are more.